common/changes/@itwin/appui-react/add-eslint-deprecation-exceptions_2023-12-20-18-38.json

```
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/appui-react",
+      "comment": "ignore appui-abstract eslint deprecation m",
```

The message is incomplete, but I dont think we need to put that in the changelogs anyway.
```suggestion
      "comment": "",
```